app/src/main/java/com/github/se/eduverse/model/Widget.kt

```diff
@@ -0,0 +1,9 @@
+package com.github.se.eduverse.model
+
+data class Widget(
+    val widgetId: String = "",
+    val widgetType: String = "",
+    val widgetTitle: String = "",
+    val widgetContent: String = "",
+    val ownerUid: String = ""
```

Comment:
I don't think the values should have a default value, otherwise in case of bad implementation we may encounter problems (e.g. many widgets with id "")

Reply:
okay, working on it now

Reply:
i could not do this because i was getting this error, i left it like this for now.

`FATAL EXCEPTION: main
                                                                                                    Process: com.github.se.eduverse, PID: 11012
                                                                                                    java.lang.RuntimeException: Could not deserialize object. Class com.github.se.eduverse.model.Widget does not define a no-argument constructor. If you are using ProGuard, make sure these constructors are not stripped
                                                                                                    	at com.google.firebase.firestore.util.CustomClassMapper.deserializeError(CustomClassMapper.java:579)`